UnityProject/Assets/Scripts/HealthV2/Living/Damage/BodyPartTraumaDamage.cs

```diff
@@ -192,6 +193,69 @@ private void CheckCutSize()
 			}
 		}
 
+		private void CheckTraumaDamageLevels()
+		{
+			//Slash
+			if(currentSlashCutDamage <= 0)
+			{
+				currentSlashDamageLevel = SlashDamageLevel.NONE;
+			}
+			if(currentSlashCutDamage >= 25)
+			{
+				currentSlashDamageLevel = SlashDamageLevel.SMALL;
+			}
+			if(currentSlashCutDamage <= 50)
```

Comment:
```suggestion
			if(currentSlashCutDamage >= 50)
```

Reply:
Do these enums have the same value for each type? If so, consider moving all of this duplicate code into a method, which each damage type can call.

Reply:
In fact, it might be better to just stick with one enum for the different damage types, with burn code handling the higher 75 level a little differently than the others, though I haven't delved too much into damage code.

Reply:
To handle the ranges, a case with IsBetween would be a lot cleaner

Reply:
```cs
switch (currentSlashCutDamage)
{
    case float n when n.IsBetween(0, 25):
        currentSlashDamageLevel = DamageLevel.NONE;
        break;
    case float n when n.IsBetween(26, 50):
        currentSlashDamageLevel = DamageLevel.SMALL;
        break;
    case float n when n.IsBetween(26, 50):
        currentSlashDamageLevel = DamageLevel.MEDIUM;
        break;
    case float n when n > 50:
        currentSlashDamageLevel = DamageLevel.LARGE;
    default:
        currentSlashDamageLevel = DamageLevel.NONE;
        Logger.Error($"Unexpected slash cut damage on: {healthMaster.GameObject}, {currentSlashDamageLevel}");
}
```
It would also allow you to handle out of range damage, like if for whatever reason you got negative damage.